integration_test/deploy-dr-buckets.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+set -ex
+
+export AWS_ACCESS_KEY_ID=$bamboo_DR_AWS_ACCESS_KEY_ID
+export AWS_SECRET_ACCESS_KEY=$bamboo_DR_AWS_SECRET_ACCESS_KEY
+export AWS_DEFAULT_REGION=$bamboo_AWS_DEFAULT_REGION
+
+#remove old files from bamboo as they throw error
+rm *.tf
+git clone --branch develop --single-branch https://github.com/nasa/cumulus-orca.git && cd integration_test
+#replace prefix with bamboo prefix variable
+sed -e 's/PREFIX/'"$bamboo_PREFIX"'/g' dr-buckets.tf.template > dr-buckets.tf
+
+if ! aws s3api head-bucket --bucket $bamboo_PREFIX-dr-tf-state;then
```

Comment:
You may want to wrap your variable for better readability. So `$bamboo_PREFIX-dr-tf-state` becomes `${bamboo_PREFIX}-dr-tf-state` That would mean if `$bamboo_PREFIX=myvariable` your bucket name would be `myvariable-dr-tf-state` In addition, you may want to make sure the `$bamboo_PREFIX` variable is populated and either set a sane default or error completely if it is not set.